src/admin/client.ts

```diff
@@ -84,15 +86,21 @@ var uiCtrl = ($scope : MainWindowScope,
               $log : ng.ILogService,
               subscriberFactory : Shared.SubscriberFactory,
               fireFactory : Shared.FireFactory) => {
-    
+
     var cancelAllFirer = fireFactory.getFire(Messaging.Topics.CancelAllOrders);
     $scope.cancelAllOrders = () => cancelAllFirer.fire(new Models.CancelAllOrdersRequest());
-                  
+
+    var cleanAllClosedFirer = fireFactory.getFire(Messaging.Topics.CleanAllClosedOrders);
```

Comment:
I added this button because now trades are always displayed forever. The user can look at all the trades opened and also at all closed trades where the final net income is displayed.

Using this button, the user can manualy clean the closed trades and "start a new cycle" anytime he/she wishes.

For example, for myself, i wakeup on the morning and see all closed trades during the night with all the net incomes. Then i clean all closed trades and go to work. Once im back in the afternoon i can examinate what new trades has been closed and the net incomes during the day. Then clean them and go to sleep. And so on every day :D

The other button of clean all orders, is because what you said, in case the market moves way too off, you can manualy clean all the historic and therefore assume the losses if any.